src/client/NetworkingClient.cpp

```diff
@@ -9,17 +9,9 @@
 #include "NetworkingClient.hpp"
 
 
-NetworkingClient::NetworkingClient() {
-    mIsRegistered = false;
-    mAcceptingNetworkGameObjectsRead = true;
-    mAcceptingDirectionRead = true;
-    mDirection = sf::Vector2f(0.f, 0.f);
-}
-
-NetworkingClient::~NetworkingClient() {
-}
-
-sf::Uint32 NetworkingClient::Start() {
+NetworkingClient::NetworkingClient()
+    :mAcceptingNetworkGameObjectsRead{true}, mAcceptingDirectionRead(true), mDirection(0.f, 0.f)
```

Comment:
what's with the {true}

Reply:
read this: https://stackoverflow.com/questions/27352021/c11-member-initializer-list-vs-in-class-initializer/27354688